saleor/graphql/core/types/common.py

```diff
@@ -339,7 +339,7 @@ class PaymentError(Error):
     code = PaymentErrorCode(description="The error code.", required=True)
     variants = NonNullList(
         graphene.ID,
-        description="List of varint IDs which causes the error.",
```

Comment:
The unnecessary change. We have `variant IDs` here.

Reply:
It's a typo fix